django/contrib/gis/gdal/raster/const.py

```diff
@@ -65,8 +65,11 @@
     16: 'GCI_YCbCr_CrBand',  # Cr Chroma, also GCI_Max
 }
 
+# GDAL virtual filesystems prefix.
+VSI_FILESYSTEM_PREFIX = '/vsi'
```

Comment:
I was wondering if we should make a whitelist of allowed vsi filesystems here, instead of this wildcard approach. I spent some time looking for a list of vsi filesystems that GDAL supports, but did not find one. Probably the supported vsi functionality also depends on the specific build of GDAL anyway.

So overall I think this is fine here and we can leave it as proposed. I just wanted to make this comment here for completeness.

Reply:
I was thinking about the same, but without any static list we cannot perform a proper validation. A wildcard approach is fine IMO, we can add a small warning in the docs, e.g.
```
.. warning:

    Rasters with names starting with `/vsi*/` will be treated as rasters from
    the GDAL virtual filesystems. Django doesn't perform any extra validation.
```

Reply:
I believe the supported `vsi*` will depend on GDAL build. For example the documentation itself states that `/vsicurl` needs GDAL built with libcurl support (https://gdal.org/user/virtual_file_systems.html#vsicurl).

Reply:
Excellent, looks like we are aligned in our assessment! Thanks for the additional insights on this.